static/js/compose.js

```diff
@@ -602,8 +602,28 @@ export function initialize() {
             available_providers.big_blue_button &&
             page_params.realm_video_chat_provider === available_providers.big_blue_button.id
         ) {
+            let meeting_name = "";
+            if (compose_state.get_message_type() === "stream") {
+                meeting_name =
+                    compose_state.stream_name() +
+                    (compose_state.topic() ? " > " + compose_state.topic() : "") +
+                    " Meeting";
```

Comment:
We should spell it as "meeting", since stream/topic names have unknown case, and "checkins Meeting" would read very oddly :).  I think this applies to all cases here.

Reply:
Also .. was this function meant to use `get_recipient_label` or something?  It seems like we extracted it and then didn't use it :)